duckduckgo/Android
# Remove dialog experiment

Task/Issue URL: https://app.asana.com/0/1125189844152671/1145698852245858

**Description**:
This PR removes the system dialog experiment introduced to be fully released. 

Note: GitHub should be showing the original files deleted and the experiment ones renamed but it doesn't :(

**Steps to test this PR**:
_Normal install_
1. After a clean install make sure you see the new UI showing the system dialog.
1. Check how the URL to be opened is no longer donttrack.us but duckduckgo.com
1. Select DDG as Always and notice how you see the confirmation screen with the hiker graphic

_Select a different browser_
1. After a clean install make sure you see the new UI showing the system dialog.
1. Select a different browser as Always.
1. Go back to the app and notice how the UI changed to show the settings screen.

_Huawei device_
1. Using a Huawei device, after a clean install you should see the settings screen (I will test this since I've got a Huawei device).
---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
